frontend/src/metabase/embedding/embedding-hub/types/embedding-checklist.ts

```diff
@@ -43,6 +43,11 @@ interface EmbeddingHubAction {
   /** Internal link to a Metabase route. */
   to?: string;
 
+  /**
+   * Click handler
+   */
+  onClick?: () => void;
```

Comment:
You could've also used the `modal` action handler that's defined above:

```diff
export type EmbeddingHubModalToTrigger =
  | { type: "add-data"; initialTab: AddDataTab }
  | { type: "new-dashboard" }
+ | { type: "embed-js", useExistingUserSession: boolean }
  | { type: "xray-dashboard" };
```

Reply:
The EmbedJS wizard modal is the `New` modal and implemented via redux actions the same way as other `New` modals. So i can't use the `local` modal as you did it